wagtail/admin/templates/wagtailadmin/shared/editing_sessions/list.html

```diff
@@ -0,0 +1,96 @@
+{% load wagtailadmin_tags i18n %}
+
+<div class="w-flex w-pr-8 w-mr-5 w-border-r w-border-border-furniture">
+    {% for session in sessions|slice:":4" %}
+        {% if session.revision_id %}
+            {% fragment as avatar %}
+                <div class="w-relative">
+                    {% avatar user=session.user classname="w-mx-[1px] w-w-7 w-h-7 w-border-2 w-border-critical-200 w-ping w-ping--critical" %}
+                    {% icon name="warning" classname="w-text-critical-200 w-w-4 w-h-4 w-absolute -w-bottom-2 w-left-2 w-z-10 w-stroke-surface-header w-shadow-white group-hover:w-hidden" %}
+                </div>
+            {% endfragment %}
+
+            {% blocktranslate trimmed with user_name=session.user|user_display_name|default:_("system") asvar saved_new_version_message %}
```

Comment:
There are cases where the "session" does not really come from an `EditingSession` object. For example, a user saved a new revision since you loaded the editor, but then they left and the session gets cleaned up. In this case, we use information from the revision object to display who the user is.

However, revisions can also be made programmatically without a user attached, in which case we handle it by displaying the user name as "system". We also have a similar handling for log actions/history for when the user is deleted (or if the action was made by system).